src/domain/synthetics/markets/useMarketTokensAPR.ts

```diff
@@ -99,20 +126,46 @@ export function useMarketTokensAPR(chainId: number): MarketTokensAPRResult {
         return acc;
       }, {} as MarketTokensAPRData);
 
-      const avgMarketsAPR = Object.values(marketTokensAPRData)
+      const marketsTokensIncentiveAprData: MarketTokensAPRData = !shouldCalcBonusApr
+        ? {}
+        : marketAddresses.reduce((acc, marketAddress) => {
+            if (!rawIncentivesStats) return acc;
+
+            const arbTokensAmount = BigNumber.from(rawIncentivesStats.rewardsPerMarket[marketAddress] ?? 0);
+            const yearMultiplier = Math.floor((365 * 24 * 60 * 60) / rawIncentivesStats.period);
+            const poolValue = getByKey(marketsInfoData, marketAddress)?.poolValueMin;
+            let incentivesApr = BigNumber.from(0);
+
+            if (poolValue && poolValue.gt(0)) {
+              incentivesApr = arbTokensAmount
+                .mul(arbTokenPrice)
+                .div(poolValue)
```

Comment:
wondering if it works correctly

`arbTokensAmount` – 18 decimals
`arbTokenPrice` – 12 decimals
`poolValue` – 30 decimals

at first glance it should return 0

Reply:
arbTokenPrice is 30 decimals, we somewhere convert it